amethyst_utils/src/ortho_camera.rs

```diff
@@ -0,0 +1,206 @@
+use amethyst_core::Axis2;
+use amethyst_core::cgmath::Ortho;
+use amethyst_core::specs::{Component, DenseVecStorage, Join, System, ReadExpect, ReadStorage, WriteStorage};
+use amethyst_renderer::{Camera, ScreenDimensions};
+
+/// `Component` attached to the camera's entity that allows automatically adjusting the camera's matrix according
+/// to preferences in the "mode" field.
+/// It tries as much as possible to adjust the camera so that the world's coordinate (0, 0) is at the bottom left and
+/// (1, 1) is at the top right of the window.
+/// You must add the `CameraNormalOrthoSystem` to your dispatcher for this to take effect.
+#[derive(Default, Debug, Clone, Serialize, Deserialize)]
+pub struct CameraNormalOrtho {
+    /// How the camera's matrix is changed when the window's aspect ratio changes.
+    /// See `CameraNormalizeMode` for more info.
+    pub mode: CameraNormalizeMode,
+}
+
+impl CameraNormalOrtho {
+    pub fn camera_offsets(&self, ratio: f32) -> (f32,f32,f32,f32) {
+        self.mode.camera_offsets(ratio)
+    }
+}
+
+impl Component for CameraNormalOrtho {
+    type Storage = DenseVecStorage<Self>;
+}
+
+/// Settings that decide how to scale the camera's matrix when the aspect ratio changes.
+#[derive(Debug, Copy, Clone, Serialize, Deserialize)]
+pub enum CameraNormalizeMode {
+    /// Using an aspect ratio of 1:1, tries to adjust the matrix values of the camera so
+    /// that the direction opposite to the stretch_direction always have a world size of 1.
+    /// If you use the default `Transform` (position = 0, 0, 0),
+    /// this means that the direction opposite to stretch_direction 
+    /// will go from 0.0 to 1.0 in world coordinates.
+    /// Scene space can be lost on the specified stretch_direction.
```

Comment:
I think these docs are somewhat confusing. How about linking to an image that visually explains how it works and maybe clearing up the text a bit?